File: 백준/Bronze/11655. ROT13/ROT13.js
const fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('');
const result = input.map((item) => {
    const targetAsciiNumber = item.charCodeAt();
    if (targetAsciiNumber >= 65 && targetAsciiNumber <= 90) {
        if (targetAsciiNumber > 77) {
            return String.fromCharCode(targetAsciiNumber - 77 + 64);
        } else {
            return String.fromCharCode(targetAsciiNumber + 13);
        }
    } else if (targetAsciiNumber >= 97 && targetAsciiNumber <= 122) {
        if (targetAsciiNumber > 109) {
            return String.fromCharCode(targetAsciiNumber - 109 + 96);
        } else {
            return String.fromCharCode(targetAsciiNumber + 13);
        }
    } else {
        return item;
    }
});

console.log(result.join(''));

File: 백준/Bronze/11655. ROT13/ROT13.py
s = input()
result = ""
for item in s:
    if item >= 'a' and item <= "z":
        value = ord(item) + 13
        if value >= 123:
            value -= 26
        result += chr(value)
    elif item >= 'A' and item <= "Z":
        value = ord(item) + 13
        if value >= 91:
            value -= 26
        result += chr(value)
    else:
        result += item
print(result)